flask-app/app.py

```diff
@@ -47,19 +54,103 @@ def proxy_request_to_brouter(parameters):
     return Response(response.content, response.status_code, headers)
 
 
+def get_regions(model):
+    db_session = DBSession()
+    return {model.__tablename__: [x.to_dict() for x in db_session.query(model).all()]}
+
+
+def get_region(model, region_id):
+    db_session = DBSession()
+    region = db_session.query(model).filter_by(id_=region_id).first()
```

Comment:
If you wish, you could change `.first()` to `.one()` which will raise an exception if zero or more than 1 objects are found and then use an error handler

Reply:
`MultipleResultsFound` in case of getting county / voivodeship would mean that there is something wrong with our db, so I left it unhandled.

Reply:
The can never be two rows with the same primary key